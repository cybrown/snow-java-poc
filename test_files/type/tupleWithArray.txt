type ATuple (int, string, string[])
