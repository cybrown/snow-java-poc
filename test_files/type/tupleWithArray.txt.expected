<SnowType>
    <Identifier>ATuple</Identifier>
    <TypeTuple>
        <Identifier>int</Identifier>
        <Identifier>string</Identifier>
        <TypeArray>
            <Identifier>string</Identifier>
        </TypeArray>
    </TypeTuple>
</SnowType>
